public/kbn_sankey_vis_controller.js

```diff
@@ -58,19 +63,19 @@ module.controller('KbnSankeyVisController', function ($scope, $element, $rootSco
       .append('g')
       .attr('transform', 'translate(0, 0)');
 
-
-
     let sankey = d3.sankey()
       .nodeWidth(15)
       .nodePadding(10)
       .size([width, height]);
 
     let path = sankey.link();
 
+    let defs = svg.append('defs');
+
     sankey
       .nodes(energy.nodes)
       .links(energy.links)
-      .layout(32);
+      .layout(13);
```

Comment:
Hi @mmguero , can you explain please why changing this parameter?

Reply:
I read several examples (such as [this one](https://bl.ocks.org/micahstubbs/3c0cb0c0de021e0d9653032784c035e9)) and tested it myself: any value > 13 seems to break the gradient (see [this example](https://bl.ocks.org/micahstubbs/bf90fda6717e243832edad6ed9f82814)). I don't know the reason behind it.